src/shared/components/plots/StackedBarPlotUtils.spec.ts

```diff
@@ -629,7 +629,7 @@ describe("StackedBarPlotUtils", ()=>{
             }], "given major order (alphabetical minor)");
 
             assert.deepEqual(makeBarSpecs(
-                multipleValueHorzBarsData, categoryOrder, categoryOrder, getColor, categoryCoord, true
+                multipleValueHorzBarsData, categoryOrder, categoryOrder, getColor, categoryCoord, true, false
```

Comment:
it would be good to add tests of data created with percentage calculation as well - ensure its values for `y` and `percentage` are correct

Im surprised these unit tests would even be passing since it looks like you're adding the `percentage` member to the data

Reply:
will add test once we are ok with styles and everything